Editor/AGS.Editor/Components/AudioComponent.cs

```diff
@@ -547,6 +553,55 @@ private bool ForceReimportOfAudioClip(AudioClip clip)
             return reimportOk;
         }
 
+        private void DoForceReimportOfAudioClips(IEnumerable<AudioClip> audioClips, IWorkProgress progress, out List<AudioClip> failedClips)
+        {
+            failedClips = new List<AudioClip>();
+            progress.Total = audioClips.Count();
+            progress.Current = 0;
+            int i = 0;
+            foreach (AudioClip clip in audioClips)
+            {
+                if (!ForceReimportOfAudioClip(clip))
+                {
+                    failedClips.Add(clip);
+                }
+                progress.Current = ++i;

```

Comment:
Would not `progress.Current++` work, without `i` variable?

Reply:
I think it would, but I saw this pattern in the other places with it and thought it was necessary for some reason.

Reply:
It looks like it works fine, just curious that it's type is `int?` instead of just `int`, probably means it can be null somehow?

Reply:
> It looks like it works fine, just curious that it's type is `int?` instead of just `int`, probably means it can be null somehow?

yes, `int?` means that it begins as null and may be assigned a null. But since you assign it with `0` earlier, this should not be a problem here.

iirc I made it nullable to indicate that the process does not feature a progress counter.